src/neo/Ledger/Blockchain.cs

```diff
@@ -298,14 +293,6 @@ protected override void OnReceive(object message)
             }
         }
 
-        private void OnTransaction(Transaction tx)
-        {
-            if (ContainsTransaction(tx.Hash))
```

Comment:
This is removed. But I think we should check it first before sending the transaction to `txrouter`.

Reply:
Maybe we can check this in RemoteNode rather than in Blockchain to avoid message hoop

Reply:
We can check it in `RemoteNode` or `TransactionRouter`.

Reply:
Have added